packages/dds/tree/src/simple-tree/api/customTree.ts

```diff
@@ -186,3 +202,30 @@ export function tryStoredSchemaAsArray(
 		}
 	}
 }
+
+/**
+ * Clones tree, replacing any handles.
+ * @remarks
+ * This can be useful converting data containing handles to JSON compatible formats,
+ * or just asserting data does not contain handles.
+ *
+ * Reversing this replacement depends on how the replacer encodes handles, and can often be impossible if the replacer
+ * does not have all the necessary context to restore the handles
+ * (e.g. if the handles are something insufficiently descriptive,
+ * if data referenced by the handle got garbage collected,
+ * if the encoded form of the handled couldn't be differentiated from other data,
```

Comment:
Nit (other clauses are present tense):
```suggestion
 * if data referenced by the handle gets garbage collected,
 * if the encoded form of the handled can't be differentiated from other data,
```

Reply:
This was intentional. When trying to restore handles occurs, the encoding, and possible GC would be in the past, but the decoder accessing the container to get the handles would be in the present.

Reply:
Fair enough. I think the feedback stands for the second of the two though.

Reply:
Also note the "handled" typo in the second, which I initially missed.

Reply:
```suggestion
 * if data referenced by the handle got garbage collected,
 * if the encoded form of the handle can't be differentiated from other data,
```